facilitator/src/bin/main.rs

```diff
@@ -0,0 +1,652 @@
+                            processor",
+                        )
+                        .long_help(
+                            "Base64 encoded ECDSA P256 public key for the peer \
+                            share processor. If not specified, a default key \
+                            will be used.",
+                        )
+                        .default_value(DEFAULT_PHA_SIGNING_PRIVATE_KEY)
+                        .hide_default_value(true)
+                        .validator(b64_validator),
+                )
+                .arg(Arg::with_name("is-first").long("is-first").help(
+                    "Whether this is the \"first\" server receiving a share, i.e., the PHA.",
+                )),
+        )
+        .get_matches();
+
+    let _verbose = matches.is_present("verbose");
+
+    match matches.subcommand() {
+        // The configuration of the Args above should guarantee that the
+        // various parameters are present and valid, so it is safe to use
+        // unwrap() here.
+        ("generate-ingestion-sample", Some(sub_matches)) => generate_ingestion_sample(
+            &mut FileTransport::new(
+                Path::new(sub_matches.value_of("pha-output").unwrap()).to_path_buf(),
+            ),
+            &mut FileTransport::new(
+                Path::new(sub_matches.value_of("facilitator-output").unwrap()).to_path_buf(),
+            ),
+            &sub_matches
+                .value_of("batch-id")
+                .map_or_else(|| Uuid::new_v4(), |v| Uuid::parse_str(v).unwrap()),
+            &sub_matches.value_of("aggregation-id").unwrap(),
+            &sub_matches.value_of("date").map_or_else(
+                || Utc::now().naive_utc(),
+                |v| NaiveDateTime::parse_from_str(&v, DATE_FORMAT).unwrap(),
+            ),
+            &PrivateKey::from_base64(sub_matches.value_of("pha-private-key").unwrap()).unwrap(),
+            &PrivateKey::from_base64(sub_matches.value_of("facilitator-private-key").unwrap())
+                .unwrap(),
+            &base64::decode(sub_matches.value_of("ingestor-private-key").unwrap()).unwrap(),
+            sub_matches
+                .value_of("dimension")
+                .unwrap()
+                .parse::<i32>()
+                .unwrap(),
+            sub_matches
+                .value_of("packet-count")
+                .unwrap()
+                .parse::<usize>()
+                .unwrap(),
+            sub_matches
+                .value_of("epsilon")
+                .unwrap()
+                .parse::<f64>()
+                .unwrap(),
+            sub_matches
+                .value_of("batch-start-time")
+                .unwrap()
+                .parse::<i64>()
+                .unwrap(),
+            sub_matches
+                .value_of("batch-end-time")
+                .unwrap()
+                .parse::<i64>()
+                .unwrap(),
+        )
+        // Drop return value
+        .map(|_| ()),
```

Comment:
Just add brackets around the case body so you can finish the call with a semicolon